src/js/components/TaskTable.js

```diff
@@ -196,7 +189,10 @@ class TaskTable extends React.Component {
 
 TaskTable.propTypes = {
   className: React.PropTypes.string,
-  parentRouter: React.PropTypes.func.isRequired,
+  parentRouter: React.PropTypes.oneOfType([
```

Comment:
Can't we just use `JestUtil.stubRouterContext`? It's based on [https://github.com/reactjs/react-router/blob/0.13.x/docs/guides/testing.md](https://github.com/reactjs/react-router/blob/0.13.x/docs/guides/testing.md)